mls-rs/src/group/mod.rs

```diff
@@ -995,6 +966,23 @@ where
         &mut self,
         message: &[u8],
         authenticated_data: Vec<u8>,
+    ) -> Result<MlsMessage, MlsError> {
+        self.encrypt_application_message_with_padding(
+            message,
+            authenticated_data,
+            PaddingMode::None,
+        )
+        .await
+    }
+
+    // TODO make a builder
+    #[cfg(feature = "private_message")]
+    #[cfg_attr(not(mls_build_async), maybe_async::must_be_sync)]
+    pub async fn encrypt_application_message_with_padding(
```

Comment:
I feel like in the future even stuff like this can be handled via the default above and some builder that deals with authenticated_data as well

Reply:
Agreed, that's what the TODO comment is about.